src/AppInstallerCommonCore/Architecture.cpp

```diff
@@ -8,30 +8,67 @@
 
 namespace AppInstaller::Utility
 {
+    namespace
+    {
+        // Gets the applicable architectures for the current machine.
+        std::vector<Architecture> CreateApplicableArchitecturesVector()
+        {
+            std::vector<Architecture> applicableArchs;
+
+            switch (GetSystemArchitecture())
+            {
+            case Architecture::Arm64:

```

Comment:
Use IsWow64GuestMachineSupported to truly determine if WOW architectures are enabled.